pkg/apikey/apikey.go

```diff
@@ -22,6 +18,7 @@ type Metadata struct {
 	Name          string   `json:"name"`
 	Organizations []string `json:"organizations"`
 	PublicKey     string   `json:"public_key"`
+	Scheme        string   `json:"scheme"`
```

Comment:
Are we using Scheme and Curve interchangably, or is the selection of a particular curve a Scheme, but also other things could be Schemes?  Are Schemes additive?  Can you aggregate multiple schemes, each with different aspects?

Reply:
> Are we using Scheme and Curve interchangably, or is the selection of a particular curve a Scheme, but also other things could be Schemes?

Scheme applies to the signature, and theoretically doesn't need to have anything to do with a curve. A signature could be RSA for example, with no curve. A curve in this case correlates to a scheme, but not necessarily the other way around.

> Are Schemes additive? Can you aggregate multiple schemes, each with different aspects?

To me, scheme correlates directly to the property `SignatureScheme` of the JSON object that is stamped. So, it's a piece of metadata that tells the stamp verifier how the stamp's signature component should be verified. So, no aggregation would be possible in any explicit sense, since signature <> scheme is 1:1 pairing.